hack/local-up-cluster.sh

```diff
@@ -410,7 +410,9 @@ EOF
 
     # Wait for kube-apiserver to come up before launching the rest of the components.
     echo "Waiting for apiserver to come up"
-    kube::util::wait_for_url "https://${API_HOST}:${API_SECURE_PORT}/version" "apiserver: " 1 ${WAIT_FOR_URL_API_SERVER} || exit 1
+    # this uses the API port because if you don't have any authenticator, you can't seem to use the secure port at all.
```

Comment:
> have enough understanding on this.

It's related.  By effectively denying all access through the secure API, the check against `/version` was always denied.